app/contextCreator/createSolutionListPageContext.js

```diff
@@ -0,0 +1,17 @@
+export const createSolutionListPageContext = (solutions, pageTitle, pageDescription) => {

```

Comment:
Going to be really picky but the order of these. Would prefer title, description and then solutions

Reply:
Yeah I agree but the below comment will change this anyway